src/catalog/src/information_schema/predicate.rs

```diff
@@ -14,10 +14,10 @@
 
 use arrow::array::StringArray;
 use arrow::compute::kernels::comparison;
-use common_query::logical_plan::DfExpr;
 use datafusion::common::ScalarValue;
 use datafusion::logical_expr::expr::Like;
 use datafusion::logical_expr::Operator;
+use datafusion::prelude::Expr;
```

Comment:
non-blocking: Maybe import Expr from logical_expr.
```suggestion
use datafusion::logical_expr::{Expr, Operator};
```